src/main/java/org/folio/circulation/domain/validation/RequestedByAnotherPatronValidator.java

```diff
@@ -0,0 +1,48 @@
+package org.folio.circulation.domain.validation;
+
+import static org.folio.circulation.support.Result.succeeded;
+
+import java.util.function.Function;
+
+import org.folio.circulation.domain.Item;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.domain.User;
+import org.folio.circulation.support.Result;
+import org.folio.circulation.support.ValidationErrorFailure;
+
+public class RequestedByAnotherPatronValidator {
+  private final Function<String, ValidationErrorFailure> errorFunction;
+
+  public RequestedByAnotherPatronValidator(
+    Function<String, ValidationErrorFailure> errorFunction) {
+
```

Comment:
Is this blank line needed?

Reply:
I personally find that when a method signature spans multiple lines, it aids readability to leave a gap to the implementation.

Do you think it would be preferable to remove it?

Reply:
That seems reasonable to me